### PR TITLE
feat: added devcontainer config for vscode

### PR DESCRIPTION
**Pull request by sainak**
This PR adds devcontainer config for vscode to allow development of care fe inside a docker container this can help reduce initial development environment overhead and errors.
Further reading: https://code.visualstudio.com/docs/remote/containers

### CODE REVIEW

**Comment by arpancodes** [comment]
@sainAk do add a good description for your PRs.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "args": {
+      "VARIANT": "16-bullseye"
+    }
+  },
+
+  // Set *default* container specific settings.json values on container create.
+  "settings": {},
+
+  // Add the IDs of extensions you want installed when the container is created.
+  "extensions": [
```
Comment (sainak) [inline .devcontainer/devcontainer.json:19]: I've added the basic extentions that are required for development but we can specify additional extensions for the container, suggestions are welcomed.

**Comment by gigincg** [comment]
@sainAk Can you fix the conflict?

**Comment by nihal467** [comment]
@gigincg @khavinshankar what is the status of the PR

**Pull request closed without merging**